tests/gui/conftest.py

```diff
@@ -0,0 +1,27 @@
+from streamlit.testing.v1 import AppTest
+from alphastats.load_data import load_data
+from pathlib import Path
+from alphastats import DataSet
+
+
+APP_FOLDER = Path(__file__).parent / Path("../../alphastats/gui/")
+TEST_INPUT_FILES = f"{APP_FOLDER}/../../testfiles"
+
+
+def print_session_state(apptest: AppTest):
```

Comment:
good idea collecting these also here!

but there are some redefinitions now that should be subsituted by these
e.g. in test_03_data_overview.py